tor/strings/en_US.yml

```
@@ -17,6 +17,8 @@ mod:
   formatting_issues: |
     I detected formatting issues in the transcription by {author}: {issues}
     {link}
+  transcript_on_tor_post: |
+    Looks like {user_url} posted a [transcription]({post_url}) on [ToR Post]({tor_post}) instead of posting it on [r/{sub_name}]({post_link}). :facepalm:
```

This string is going to slack, right? It will work better if we use the native slack link formatting.

```suggestion
    Looks like <{user_url}|{user_url}> posted a <{post_url}|transcription> on <{tor_post}|the ToR Post> instead of posting it on <{post_link}|r/{sub_name}>. :facepalm:
```

---

Good catch!

---

alright I finally found some more time to work on this. I didn't know that slack had some different formatting for that! Thankss